### ISSUE TITLE
Consider displaying `dataPath` for element templates validation errors

### ISSUE THREAD

**Issue by pinussilvestrus** [bpmn-io/bpmn-js-properties-panel#459]
__Is your feature request related to a problem? Please describe__

Starting with #455 we would potentially have a `dataPath` for each error we display in the Log Panel. The data path is a direct pointer to the place where the validation error was detected. We could potentially display it if we want to (cf. screenshot).

![Bildschirmfoto 2021-03-26 um 08 45 11](https://user-images.githubusercontent.com/9433996/112602194-a80ed800-8e13-11eb-86cb-2a8dfbdc9abc.png)

It adds another good indicator where a validation error was detected. Example

> What I could read from the data path: it is the third (fourth if count starts from 0) input parameter that has the error. Otherwise, I would have to guess which input parameter causes the error. 

__Describe the solution you'd like__

Print the `dataPath`.

__Describe alternatives you've considered__

Why we shouldn't do it:
* It blows out the error message a bit more
* They could be cryptic in some cases, e.g. `/scopes/camunda:Connector/properties/1/type`
* The error messages shall be clear enough

Another alternative could be to display a line number instead of the path (cf. https://github.com/camunda/element-templates-json-schema/issues/4).

**Comment by pinussilvestrus** [comment]
I tried out whether / how we could get the line of a given error to better display the location. Basically saying, it would be possible. Take this branch in `json-schema-validator`: https://github.com/bpmn-io/json-schema-validator/compare/459-data-pointer?expand=1

This would set a `dataPointer` next to the `dataPath` for each error, which looks something like that:

```js
{
  keyword: 'errorMessage',
  ...
  dataPath: '/properties/0/name',
  dataPointer: {
    key: { line: 5, column: 6, pos: 97 },
    keyEnd: { line: 5, column: 12, pos: 103 },
    value: { line: 5, column: 14, pos: 105 },
    valueEnd: { line: 5, column: 19, pos: 110 }
  }
}
```

For a given `dataPath` for an error, it will also set the exact location (line number, column, position). This has two problems:
* We set the data path + data pointer inside the scope of a single object/template.
* We only validate one template at once, which makes it extremely hard to find the error back in a set of multiple templates via line number. 

==> Getting the line number is by far not trivial. However, we could add the data pointer to our library, since for single errors it will add a notable benefit (more information about errors.

I'd still not go for displaying the raw dataPath, now. Yes, it will help in some cases, but can be cryptic in huger templates. I'd rather display less instead of wrong or misunderstood information.

**tldr: I would**
* add the `dataPointer` to our validation library as new feature, **but**
* keep things and not display dataPath + dataPointer (for now)

**Comment by pinussilvestrus** [comment]
`ajv` actually shipped a new release `v8` which changes the term `dataPath` to `instancePath`: https://github.com/ajv-validator/ajv/releases/tag/v8.0.0

🤡

**Comment by nikku** [comment]
@philippfromme Sounds like they have the same discussions we had in the form territory. Instance, schema, waaaat? :hear_no_evil:
